src/main/java/io/javalin/core/JettyServerUtil.kt

```diff
@@ -22,20 +20,26 @@ import org.eclipse.jetty.servlet.ServletContextHandler
 import org.eclipse.jetty.servlet.ServletHolder
 import org.eclipse.jetty.websocket.servlet.WebSocketServlet
 import org.eclipse.jetty.websocket.servlet.WebSocketServletFactory
-import org.slf4j.LoggerFactory
+import org.slf4j.Logger
 import java.io.ByteArrayInputStream
 import javax.servlet.http.HttpServletRequest
 import javax.servlet.http.HttpServletResponse
 
-class EmbeddedJettyServer(private val server: Server, private val javalinServlet: JavalinServlet) : EmbeddedServer {
+object JettyServerUtil {
 
-    private val log = LoggerFactory.getLogger(EmbeddedServer::class.java)
+    @JvmStatic
+    fun initialize(
```

Comment:
should you perhaps test this function on its own? or do you have enough tests of the Javalin.start()?

Reply:
It's being tested every time the server starts, and the server starts for almost every test.